Sprint-1/exercises/paths.js

```diff
@@ -1,18 +1,22 @@
-// The diagram below shows the different names for parts of a file path on a Unix operating system
-
-// ┌─────────────────────┬────────────┐
-// │          dir        │    base    │
-// ├──────┬              ├──────┬─────┤
-// │ root │              │ name │ ext │
-// "  /    home/user/dir / file  .txt "
-// └──────┴──────────────┴──────┴─────┘
-
-// (All spaces in the "" line should be ignored. They are purely for formatting.)
-
-const filePath = "/Users/mitch/cyf/Module-JS1/week-1/interpret/file.txt";
-const lastSlashIndex = filePath.lastIndexOf("/");
-const base = filePath.slice(lastSlashIndex + 1);
-console.log(`The base part of ${filePath} is ${base}`);
-
-// Create a variable to store the dir part of the filePath variable
-// Create a variable to store the ext part of the variable
+// The diagram below shows the different names for parts of a file path on a Unix operating system
+
+// ┌─────────────────────┬────────────┐
+// │          dir        │    base    │
+// ├──────┬              ├──────┬─────┤
+// │ root │              │ name │ ext │
+// "  /    home/user/dir / file  .txt "
+// └──────┴──────────────┴──────┴─────┘
+
+// (All spaces in the "" line should be ignored. They are purely for formatting.)
+
+const filePath = "/Users/mitch/cyf/Module-JS1/week-1/interpret/file.txt";
+const lastSlashIndex = filePath.lastIndexOf("/");
+const base = filePath.slice(lastSlashIndex + 1);
+console.log(`The base part of ${filePath} is ${base}`);
+
+// Create a variable to store the dir part of the filePath variable
+const dirpart=filePath.slice(0,lastSlashIndex);
+console.log(`the dir part of ${filePath} is : ${dirpart}`);
+// Create a variable to store the ext part of the variable
+const lastdotindex=filePath.lastIndexOf(".");

```

Comment:
The exercise is asking us to store the "ext" part of the `filePath` variable. Are we really storing the "ext" part here? What do you see when you run the code?

Reply:
sorry @moneyinthesky ,
I misunderstood. Now it's been amended to show "ext" part of the filePath.
Thanks.